data/tools/converters.yml

```diff
@@ -384,3 +388,279 @@
+        -   EEG
+        -   MEG
+        -   iEEG
+        -   behavioral
+        -   MRI (anat and func)
+        expected_input:
+        -   any EEG or MEG file format
+        -   NBS Presentation logfiles
+        -   DICOM
+        -   nifti
+        language:
+        -   MATLAB
+        documentation: https://www.fieldtriptoolbox.org/example/bids/
+        license: GPL-3.0
+
+    -   name: Biscuit
+        url: https://github.com/Macquarie-MEG-Research/Biscuit
+        data_types:
+        -   MEG
+        expected_input:
+        -   ???
+        language:
+        -   Python
+        documentation: https://macquarie-meg-research.github.io/Biscuit/
+        comment: GUI for easy MEG to BIDS conversion
+        GUI: true
+        license: MIT
+
+    -   name: sovabids
+        url: https://github.com/yjmantilla/sovabids
+        data_types:
+        -   EEG
+        expected_input:
+        -   ???
+        language:
+        -   Python
+        documentation: https://sovabids.readthedocs.io/en/latest/
+        comment: A Python package for the automatic conversion of EEG datasets to the BIDS standard, with a focus on making the most out of metadata.
+
+    -   name: EEG2BIDS
+        url: https://github.com/aces/EEG2BIDS
+        data_types:
+        -   EEG
+        expected_input:
+        -   ???
+        language:
+        -   Javascript
+        -   Python
+        documentation:
+        comment: A tool for converting raw EEG and iEEG data into the BIDS standard data structure, prepared for LORIS (Longitudinal Online Research and
+            Imaging System).
+
+
+# Physiological data converters
+    -   name: phys2bids
+        url: https://github.com/physiopy/phys2bids
+        data_types:
+        -   phsyiological
+        expected_input:
+        -   Acqknowledge
+        -   Labchart
+        -   GE physiological files
+        language:
+        -   Python
+        distribution:
+        -   name: pypi
+            url: https://pypi.org/project/phys2bids
+        documentation: https://phys2bids.readthedocs.io/en/latest/
+        comment: Python3 library to format physiological files in BIDS.
+        license: Apache-2.0
+
+    -   name: bidsphysio
+        url: https://github.com/cbinyu/bidsphysio
+        data_types:
+        -   physiological
+        expected_input:
+        -   CMRR
+        -   AcqKnowledge
+        -   Siemens PMU
+        distribution:
+        -   name: pypi
+            url: https://pypi.org/project/bidsphysio/
+        -   name: dockerhub
+            url: https://hub.docker.com/r/cbinyu/bidsphysio/
+        language:
+        -   Python
+        documentation:
+        comment: Converts physio data to BIDS physiological recording
+        GUI: false
+
+
+# 'From BIDS' converters
+# Comment: Converters that take a BIDS dataset as input to convert it into something else. 
+# Not mentioned here are the many software that can import a BIDS data as data structure 
+# they are more familiar with.
```

Comment:
I think you can remove those comments from here and add them in the markdown after the heading that corresponds to those converters.